### ISSUE TITLE
Seared faucet and Smeltery controller uncraftable

### ISSUE THREAD

**Issue by BinaryEnigma01** [al132mc/antimatter-chemistry#227]
I can't craft either of them ( JEI shows no recipe and the default recipe doesn't work), I think they were supposed to be unlocked when I completed the "Gather Ores" quest, seeing as it has "More Recipes Unlocked" as a reward. This means I'm stuck at the "Smelting" quest, I've crafted every other item necessary

**Comment by ghost** [comment]
I'm still not quite sure why gamestages seem to malfunction so frequently.. but the easiest way to fix this would be to enter the command ``/gamestage add @p tinkers``

**Comment by BinaryEnigma01** [comment]
Wow, thanks for the ridiculously fast reply, it worked!

**Comment by BinaryEnigma01** [comment]
It happened again with the scarab crafting recipe, additional info: On a server owned by me, this time noticed that when I claim the recipe the console logs "Unknown command"
(Also what's the name of this gamestage?)

**Comment by ghost** [comment]
This gamestage is just called ``energy`` i.e. ``/gamestage add @p energy``. I'm not sure why this bug is happening assuming you aren't running a cracked client, but it seems to happen to 1 in every few hundred people.

**Comment by ghost** [comment]
There is one more gamestage in the second to last chapter if this comes up again. The command is ``/gamestage add @p netherstar``

**Comment by ghost** [comment]
Actually. there is yet another game stage (4 in total, all mentioned now in this thread).
``/gamestage add @p mummy`` in chapter 4

**Comment by BinaryEnigma01** [comment]
My account is premium (if we still call it that) and my server is set to online mode, also, I'm guessing the server itself counts as 1 out of those 100 people, considering another player on my team reported the same. I'm hoping this will be fixed eventually, until then I may just add my own reward for the quest and a shop at spawn that trades it for the command.
Thank you for the temp fix, please open this thread again if you ever find out how to fix it more permanently

**Comment by BinaryEnigma01** [comment]
I figured it out. I went into editing mode to see the quest rewards to see that it's the commands you gave me (for some reason I thought it would be some other command), and it has a `/` at the start, as it turns out, `/say hi` returns "Unknown Command" while `say hi` makes the [Server] say "hi" (I thought `/` also worked in servers).
I removed the `/` from all the gamestage quests, this should only work for servers, and thus the original version should only fail for certain servers. 
Thanks for the help and for pointing me in the right direction
